project/AppDependencies.scala

```diff
@@ -24,7 +24,7 @@ object AppDependencies {
     ws,
     "uk.gov.hmrc" %% "bootstrap-backend-play-27" % "2.20.0",
     "uk.gov.hmrc"   %% "domain"            % "5.10.0-play-27",
-    "org.typelevel" %% "cats-core"         % "2.2.0",
+    "org.typelevel" %% "cats-core"         % "2.3.0",
     compilerPlugin("com.github.ghik" % "silencer-plugin" % "1.7.1" cross CrossVersion.full),
     "com.github.ghik" % "silencer-lib" % "1.7.1" % Provided cross CrossVersion.full,
     "com.chuusai"   %% "shapeless"         % "2.3.3"
```

Comment:
This should be 2.4.0-M1

Reply:
That release tag doesn't look like a main release, generally safer to stick to main releases/versions

Reply:
This is what we are using in most of other services so I think this one is safe to be bumped to that 2.4.0-M1. Example is losses in prod [here](https://github.com/hmrc/individual-losses-api/blob/master/project/AppDependencies.scala#L31)

Reply:
I don't think that's your call to make. They may have been upgraded by mistake. If we do go ahead with 2.4.0-M1, then it's better to get consensus from the team

Reply:
Not saying it is my call, just saying it is safe to do so as it is done for most of our services. If we do want to change, then I guess your other PRs which use 2.4.0-M1 currently and other services will need this changing